.gitignore

```diff
@@ -0,0 +1,57 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
```

Comment:
https://golang.org/ref/mod#vendoring 
remove the comment above to include it.

Reply:
oh, thanks, missed it out